pkg/cryptutil/certificates.go

```diff
@@ -37,6 +38,24 @@ func CertificateFromFile(certFile, keyFile string) (*tls.Certificate, error) {
 	return &cert, err
 }
 
+// CRLFromBase64 parses a certificate revocation list from a base64 encoded blob.
+func CRLFromBase64(rawCRL string) (*pkix.CertificateList, error) {
+	bs, err := base64.StdEncoding.DecodeString(rawCRL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode base64 crl: %w", err)
+	}
+	return x509.ParseCRL(bs)
```

Comment:
_ParseCRL parses a CRL from the given bytes. It's often the case that PEM encoded CRLs will appear where they should be DER encoded, so this function will transparently handle PEM encoding as long as there isn't any leading garbage._

maybe we should explicitly decode PEM block and complain if it's not the right format. i.e. if we would allow uploading it from console.

Reply:
I updated the function to only allow PEM encoded CRLs.